core/src/c_api/tiledb_utils.cc

```diff
@@ -281,7 +281,12 @@ int create_temp_filename(char *path, size_t path_length) {
   }
   char tmp_filename_pattern[64];
   sprintf(tmp_filename_pattern, "%s/TileDBXXXXXX", tmp_dir);
+#ifndef __APPLE__
   int tmp_fd = mkostemp(tmp_filename_pattern, O_APPEND|O_CLOEXEC|O_SYNC);
+#else
+  //Why are the extra flags needed? Older versions of OSX do not have mkostemp
+  int tmp_fd = mkstemp(tmp_filename_pattern);
+#endif
```

Comment:
@nalinigans - is this ok? Looks like OSX 10.9 does not contain mkostemp

Reply:
FLAGS are not strictly necessary. So, go ahead and just use mkstemp for all platforms.

Reply:
That's what Broad wanted when we started - I don't know now